src/main/java/fr/xephi/authme/listener/PlayerListener.java

```diff
@@ -376,6 +382,34 @@ public void onPlayerRespawn(PlayerRespawnEvent event) {
         if (spawn != null && spawn.getWorld() != null) {
             event.setRespawnLocation(spawn);
         }
+
+        if (settings.getProperty(RestrictionSettings.SPECTATE_STAND_LOGIN)
+            || spectateLoginService.hasStand(event.getPlayer())) {
+            // If a player is dead, no stand will be created for him
+            // therefore we can be sure that there will not be two stands
+            bukkitService.runTaskLater(() -> spectateLoginService.createStand(event.getPlayer()), 1L);
```

Comment:
I can't confirm the comment about stands not being created if a player is dead but maybe I'm missing the relevant point, but interestingly the logic here says if the setting is enabled, _or_ the player has a stand, then a stand is created? I don't think that logic adds up

Reply:
Now an armor stand is created for a player every time he respawns, regardless of whether an armor stand already exists or not

Reply:
Done.